src/doc/building_visit/Linux_X11_Qt.rst

```diff
@@ -9,13 +9,10 @@ Many developers encounter errors building Qt due to missing support libraries re
 Here are a couple of Qt links that prove helpful in knowing which packages need to be installed for particular OS distros.
 
 * For Qt 5
-
-  * The libxcb section of `Building Qt5 from Git <https://wiki.qt.io/Building_Qt_5_from_Git>`_
-
-  * Qt5's `X11 Requirements <https://doc.qt.io/qt-5/linux-requirements.html>`_ page.
+  * The libxcb section of `Building Qt5 from Git <https://wiki.qt.io/Building_Qt_5_from_Git>`_  
```

Comment:
The blank lines are needed for this nested bulleted section.
```suggestion

  * The libxcb section of `Building Qt5 from Git <https://wiki.qt.io/Building_Qt_5_from_Git>`_  
  
```

Reply:
Am rendering locally and getting warnings. I think I fixed and pushed a commit. We'll see how it renders on RTD in a min.